src/home/secondary-features/space.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+export const Space = () => <Grid item xs={12} sm={12} md={12} lg={1} xl={1} />
```

Comment:
To avoid this you can use justifyContent: 'space-between' on the parent container.

Reply:
Same here. It will break the md breakpoint. Text won't be centered anymore.

![image](https://user-images.githubusercontent.com/39530866/123602997-0fb53480-d802-11eb-8a31-147fecd4574e.png)